Restaurant page completion

ershadul1 opened the pull request:
### What does this PR do?

- [X] Uses webpack to bundle source files.
- [X] Creates three tabs in a single-page app.
- [X] Makes a responsive website
- [X] Uses the concepts of modular code in javascript.
- [X] Follows coding best practices.

ershadul1 committed:
Add starter files

ershadul1 committed:
Add webpack config

ershadul1 committed:
Add .vscode in gitignore

ershadul1 committed:
Remove .vscode

ershadul1 committed:
Delete settings.json

ershadul1 committed:
Add basic styles for flex-box

ershadul1 committed:
Fix stylelint error

ershadul1 committed:
Add reset css

ershadul1 committed:
Merge branch 'restaurant-page' of github.com:ershadul1/js-restaurant-page into restaurant-page

ershadul1 committed:
Add bare bone for the project

ershadul1 committed:
Add webpack config for images

ershadul1 committed:
Add general styles

ershadul1 committed:
Add styles to navbar

ershadul1 committed:
Add styles in home page

ershadul1 committed:
Build using webpack

ershadul1 committed:
Add contents to menu page

ershadul1 committed:
Add contents in about page

ershadul1 committed:
Add footer contents

ershadul1 committed:
Fix linter errors

ershadul1 committed:
Update README.md

ershadul1 committed:
Update README.md

lucasmazo23 commented on src/js/tab-switcher.js:
```diff
@@ -0,0 +1,27 @@
+      innerContainer.appendChild(aboutPage);
+      break;
+  }
+}
```
- [x] [REQUIRED] Each module should be able to append itself to the DOM
![image](https://user-images.githubusercontent.com/73254040/99142438-88bc9e80-261a-11eb-8f3e-1ace27fd9327.png)
[Source](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)

ershadul1 replied:
Fixed the code so that each module can append itself to the DOM.

lucasmazo23 commented on src/js/tab-switcher.js:
```diff
@@ -0,0 +1,27 @@
+import menuPage from './menu'
+import aboutPage from './about'
+
+function removeCurrentTab() {
```
- [x] [REQUIRED] Please  use arrow functions instead of the `function` keyword (this is only an example, all `function` keywords should be replaced)
![image](https://user-images.githubusercontent.com/73254040/99142509-1bf5d400-261b-11eb-95c4-53775e232778.png)
[Source](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/59622983/)

ershadul1 replied:
Converted all traditional functions into arrow functions.

lucasmazo23 commented on package.json:
```diff
@@ -0,0 +1,38 @@
+  "version": "1.0.0",
+  "description": "A single page app which shows menu, about and homepage of a restaurant. The app is build using javascript.",
+  "main": "index.js",
+  "scripts": {
```
- [x] [REQUIRED] Please add the necessary run common commands like `build`, `development`
![image](https://user-images.githubusercontent.com/73254040/99142547-a5a5a180-261b-11eb-96f8-24292ee80b76.png)
[Source](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/59622983/)

ershadul1 replied:
Added build and development scripts in package.json

lucasmazo23 reviewed (changes requested):
## Changes required 🔧
### You are doing a great job so far, and the project is looking good.
 
### For the project to get approved kindly implement these changes:
 
### Required changes:
  Check my comments below to see the changes.
### Happy coding! 💻
## Lucas Mazo

ershadul1 committed:
Convert function expressions to arrow functions

ershadul1 committed:
Make the tabs modular

ershadul1 committed:
Update README.md

ershadul1 committed:
Add build and development scripts in package.json

ershadul1 committed:
Fix linter errors

ershadul1 commented:
### Changes made
Thank you very much for your thorough review @lucasmazo23. I have implemented the changes you asked for. 

- [X] Fixed the code so that each module can append itself to the DOM.
- [X] Converted all traditional functions into arrow functions.
- [X] Added build and development scripts in package.json.

lucasmazo23 reviewed (approved):
## Project approved 🚀
### Great job!
### Your project looks great!
### You can go ahead and merge this pull request!
### Happy coding! 💻
## Lucas Mazo
